schema/schemaExaminerWorkSchedule.json

```diff
@@ -183,11 +193,7 @@
     "name": {
       "description": "Details of the individual's name",
       "type": "object",
-      "required": [
```

Comment:
What's the reason for removing these required fields?

Reply:
To keep a consistent schema across the repos - they aren't required in data migration so we have made them no longer required here.